app/src/main/res/xml/grayscale_icon_map.xml

```diff
@@ -408,7 +413,6 @@
     <icon drawable="@drawable/neo_store" package="com.machiav3lli.fdroid" name="Neo Store" />
     <icon drawable="@drawable/netflix" package="com.netflix.mediaclient" name="Netflix" />
     <icon drawable="@drawable/netzclub" package="com.telefonica.netzclub.csc" name="Netzclub" />
-    <icon drawable="@drawable/newpipe" package="org.schabi.newpipe" name="NewPipe" />
```

Comment:
Any reason for removing `org.schabi.newpipe`?

Reply:
I think it was a duplicate, and I don't think that's the part of the latest commit though, it could be the ones before that

Reply:
Its still part of the PR. Do you mind re-adding it?

Reply:
Yeah I don't remember deleting it. It's back on the latest commit though, keeping that in.

Reply:
> Its still part of the PR. Do you mind re-adding it?

And by latest commit I meant the ones in this PR in my end, not the last PR.